### PR TITLE
Mw 188/create tests image loader

### PR DESCRIPTION
**Pull request by AKiroi**
- create tests to Icon Component
- create tests to Loader Component

### CODE REVIEW

File: mw-webapp/src/component/image/Image.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      />,
+    );
+  });
+
+  it("should render image component correctly", () => {
+    cy.get(getDataCy(IMAGE_CY)).should("exist");
+  });
+
+  it("should display a default image if src is not provided", () => {
+    cy.get(getDataCy(IMAGE_CY)).should("have.attr", "src").and("not.be.empty");
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.cy.tsx:25]: Please explain what is default image should render if src is not provided? Is it our image?

File: mw-webapp/src/component/image/Image.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  });
+
+  it("should display the image with correct alt text", () => {
+    cy.get(getDataCy(IMAGE_CY)).should("have.attr", "alt", LOGO_TEXT);
+  });
+
+  it("image component should be visible", () => {
+    cy.get(getDataCy("image")).should("be.visible");
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.cy.tsx:33]: PLease tell me about difference .should("exist") and .should("be.visible") methods)

File: mw-webapp/src/component/loader/Loader.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  it("should have the alt text", () => {
+    cy.get(getDataCy(LOADER_CY)).find("img").should("have.attr", "alt", ALT_TEXT);
+  });
+
+  it("should have the correct src attribute for the image", () => {
+    cy.get(getDataCy(LOADER_CY))
+      .find("img")
+      .should("have.attr", "src")
+      .and("include", "mastersWayLogo.svg");
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/loader/Loader.cy.tsx:32]: If we will change Logo image test will be broke

File: mw-webapp/src/component/image/Image.cy.tsx (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/component/image/Image.cy.tsx:33]: The main difference between these checks is that the first one verifies only the presence of the element in the DOM, while the second one verifies the visibility of the element on the page.

File: mw-webapp/src/component/loader/Loader.cy.tsx (hunk 3 above)
Comment (AKiroi) [reply mw-webapp/src/component/loader/Loader.cy.tsx:32]: yes, the test will be broke. Maybe it not a good test, removed it

File: mw-webapp/src/component/image/Image.cy.tsx (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/component/image/Image.cy.tsx:25]: it's not a correxct test, i want to create test like "displays the alt text if the image is not displayed", removed it, anr create a new test "displays the alt text if the image is not displayed

**Review by tritonJS826 — APPROVED** [review]
merge after Ekaterina's approve

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**